### PR TITLE
Update webjob packages

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -192,9 +192,9 @@
...
 "Microsoft.Azure.Management.StoragePool","1.0.0","","Microsoft.Azure.Management.StoragePool","","NA","NA","NA","","false","","","","","",""
 "Microsoft.Azure.Management.VideoAnalyzer","","1.0.0-beta.1","Microsoft.Azure.Management.VideoAnalyzer","","NA","NA","NA","","false","","","","","",""
-"Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents","","1.0.0-beta.1","Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents","","NA","NA","NA","","false","","","","","",""
+"Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents","","1.0.0-beta.1","Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents","","NA","NA","NA","client","true","","","active","","",""
 "Microsoft.Azure.SignalR.Emulator","","1.0.0-preview1-10756","Microsoft.Azure.SignalR.Emulator","","NA","NA","NA","","false","","","","","",""
-"Microsoft.Azure.WebJobs.Extensions.WebPubSub","","1.0.0-beta.1","Microsoft.Azure.WebJobs.Extensions.WebPubSub","","NA","NA","NA","","false","","","","","",""
+"Microsoft.Azure.WebJobs.Extensions.WebPubSub","","1.0.0-beta.1","Microsoft.Azure.WebJobs.Extensions.WebPubSub","","NA","NA","NA","client","true","","","active","","",""
```
Comment (JoshLove-msft) [inline _data/releases/latest/dotnet-packages.csv:197]: I didn't really know what to make of the discrepancy between the set of 3 NAs here and only 2 for those below. Should they all just be removed?

Comment (JoshLove-msft) [reply _data/releases/latest/dotnet-packages.csv:197]: These correspond to code repo/MS docs/GH docs

File: _data/releases/latest/dotnet-packages.csv [hunk 2]
```diff
@@ -158,7 +158,7 @@
 "Azure.Communication.Calling","","1.0.0-beta.28","Azure.Communication.Calling","","NA","NA","NA","","false","","","","","",""
 "Azure.Communication.NetworkTraversal","","1.0.0-beta.1","Azure.Communication.NetworkTraversal","","NA","NA","NA","","false","","","","","",""
 "Azure.IoT.TimeSeriesInsights","","1.0.0-beta.1","Azure.IoT.TimeSeriesInsights","","NA","NA","NA","","false","","","","","",""
-"Azure.Messaging.WebPubSub","","1.0.0-beta.1","Azure.Messaging.WebPubSub","","NA","NA","NA","","false","","","","","",""
+"Azure.Messaging.WebPubSub","","1.0.0-beta.1","Azure.Messaging.WebPubSub","","NA","NA","NA","client","true","","","active","","",""
```
Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:161]: Please see this line for naming pattern to follow:

https://github.com/Azure/azure-sdk/pull/2901/files#diff-2504e1e6e9b885ea4c4ba842303ca2fb716f2ca840cc2b0000a8cd50bf97331bR3

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (jongio) [reply _data/releases/latest/dotnet-packages.csv:197]: Run the site locally or in GH codespaces, see README for instructions.  Make sure links are all correct.

File: _data/releases/latest/dotnet-packages.csv (hunk 2 above)
Comment (JoshLove-msft) [reply _data/releases/latest/dotnet-packages.csv:161]: This line doesn't match the other T2 libraries, e.g. https://github.com/Azure/azure-sdk/pull/2901/files#diff-2504e1e6e9b885ea4c4ba842303ca2fb716f2ca840cc2b0000a8cd50bf97331bR42 doesn't have "NA" for any values. It seems like including NA will prevent the automation from picking up the default doc links. I will test it out and see how it looks after removing.

**Comment by weshaggard** [comment]
A few questions:

1. Why are you updating these manually? In general these fields get updated based on the release items in our DevOps and I suspect these items are currently missing there. So we should be running the Prepare-Release on these packages to get the correct data.
2. Is this a problem for other languages as well? Can we normalize the display and service names for these packages across languages.
3. When you mark a library New=true then the RepoPath cannot be NA, it should be the service directory assuming it is in the mono repo and follows the conventions and if it doesn't follow the conventions I'd like to understand as all the new libraries should.

**Comment by JoshLove-msft** [comment]
> A few questions:
> 
> 1. Why are you updating these manually? In general these fields get updated based on the release items in our DevOps and I suspect these items are currently missing there. So we should be running the Prepare-Release on these packages to get the correct data.

I wasn't sure what drove the automation. I just ran:
```.\Prepare-Release.ps1 Microsoft.Azure.WebJobs.Extensions.ServiceBus```
and I do not see a corresponding Devops work item. 

> 2. Is this a problem for other languages as well? Can we normalize the display and service names for these packages across languages.

These packages are .NET only.

> 3. When you mark a library New=true then the RepoPath cannot be NA, it should be the service directory assuming it is in the mono repo and follows the conventions and if it doesn't follow the conventions I'd like to understand as all the new libraries should.

So it should be blank rather than NA, right?

**Comment by weshaggard** [comment]
1. https://dev.azure.com/azure-sdk/Release/_workitems/edit/7854/ that is the release item that was created/updated when you ran the Prepare-Release. I see that it is currently marked as "new library"= false in that release item. We should be sure we actually want to treat these extensions libraries as new libraries by default our script doesn't set that to true unless the package starts with "Azure" [see](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/scripts/Language-Settings.ps1#L43). And in which case our automation doesn't update the csv for libraries that aren't new.

2. We might not have webjobs for other languages but we do have WebPubSub, which you seem to be updating here.

3.  For the doc links they are usually NA or blank but for the repo link we need the servicedirectory there in order to produce the correct path.

**Comment by JoshLove-msft** [comment]
> 1. https://dev.azure.com/azure-sdk/Release/_workitems/edit/7854/ that is the release item that was created/updated when you ran the Prepare-Release. I see that it is currently marked as "new library"= false in that release item. We should be sure we actually want to treat these extensions libraries as new libraries by default our script doesn't set that to true unless the package starts with "Azure" [see](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/scripts/Language-Settings.ps1#L43). And in which case our automation doesn't update the csv for libraries that aren't new.

Ah I didn't see that in the all-up query, but I see that the work item number it is output from the script itself. It should be New because we are updating these packages to follow T2 guidelines and use Azure.Core. The naming is staying the same as it currently is, so that is probably tripping up a lot of the automation.

> 2. We might not have webjobs for other languages but we do have WebPubSub, which you seem to be updating here.

Yes, WebPubSub should have an SDK for all languages. I assume this should be updated through the same process of running the release script, so I think will close this PR.

> 3. For the doc links they are usually NA or blank but for the repo link we need the servicedirectory there in order to produce the correct path.

This should be handled through the workitem automatically as well I think.

**Comment by weshaggard** [comment]
Yes my recommendation is to close this issue and make sure the workitems are updated correctly and then the automation should fix these entries.

**Comment by JoshLove-msft** [comment]
Closing.. will update work items instead.

**Pull request closed without merging**